deploy/2_configure.ts

```diff
@@ -0,0 +1,100 @@
+import { BigNumber } from "ethers";
```

Comment:
Javascript file naming conventions use a dash `-` rather than an underscore.
We could make these names a bit clearer too something like `1-deploy-oracles` `2-configure-feeds`